### PR TITLE
📝(readme) switch from minishift to oc cluster

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

The quick start guide is not up-to-date, it must be rewritten considering latest development and usage.

## Proposal

`oc cluster` is easier to configure and a more lightweight solution to work locally compared to `minishift`. The quick start documentation has been updated consequently.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,13 +72,65 @@ REPOSITORY          TAG                 IMAGE ID            CREATED
...
+1. Make sure that the `/etc/docker/daemon.json` file contains at list
+   `172.30.0.0/16` as insecure registry:
+
+```json
+{
+  "insecure-registries": ["172.30.0.0/16"]
+}
+```
+
+2. To run ES (you'll probably have an application that will use it), you will
```
Comment (sampaccoud) [inline README.md]: s/ES/Elasticsearch

File: README.md [hunk 2]
```diff
@@ -65,50 +72,111 @@ REPOSITORY          TAG                 IMAGE ID            CREATED
...
-in a particular environment with a new helper:
+```json
+{
+  "insecure-registries": ["172.30.0.0/16"]
+}
+```
 
-> When running this command, you'll be asked for a **vault password**. The
-> default value for this demo is: `arnold`.
+2. To run ES (you'll probably have an application that will use it), you will
```
Comment (sampaccoud) [inline README.md]: s/ES/Elasticsearch

File: README.md [hunk 3]
```diff
@@ -65,50 +72,111 @@ REPOSITORY          TAG                 IMAGE ID            CREATED
...
-| staff    | staff    | staff@example.com   | yes      | no           |
-| admin    | admin    | admin@example.com   | yes      | yes          |
+| :------- | :------- | :------------------ | :------: | :----------: |
+| student  | student  | student@example.com |    no    |      no      |
+| teacher  | teacher  | teacher@example.com |    no    |      no      |
+| staff    | staff    | staff@example.com   |   yes    |      no      |
+| admin    | admin    | admin@example.com   |   yes    |     yes      |
+
+_nota bene_: you will find URLs of the studio or lms services from the web
+console (Application > Routes). You should see a page like the above screenshot.
```
Comment (sampaccoud) [inline README.md]: above or below?

File: README.md [hunk 4]
```diff
@@ -65,50 +72,111 @@ REPOSITORY          TAG                 IMAGE ID            CREATED
...
+respectively):
 
 ```bash
 $ bin/bootstrap -e "env_type=staging customer=campus"
 ```
 
-If you want to run a new deployment for this project, there is also a helper for
-that:
+If you want to run a new deployment for the default project / environment, there
+is also a helper for that:
```
Comment (sampaccoud) [inline README.md]: Not very helpful instruction. I have the feeling that before you can run this command, you have to understand a bit more than that... we must explain blue/green and it must come with instructions about the "switch" command. Should we leave this for the detailed documentation and remove it from the README?

File: README.md (hunk 3 above)
Comment (jmaupetit) [reply README.md]: Arf below Thx.

File: README.md (hunk 4 above)
Comment (jmaupetit) [reply README.md]: And... this is not true, as we always deploy to next if no switch has been made.

File: README.md [hunk 5]
```diff
@@ -65,61 +72,109 @@ REPOSITORY          TAG                 IMAGE ID            CREATED
...
+
+_nota bene_: you will find URLs of the studio or lms services from the web
+console (Application > Routes). You should see a page like the screenshot below.
+
+![OpenShift routes](./docs/images/os-web-console-routes.png)
+
+### Going further
+
+By following this quick start, you only scratched the surface of Arnold's
+capabilities. We invite you to read the project's documentation (see bellow), to
```
Comment (sampaccoud) [inline README.md]: s/bellow/below

**Review by sampaccoud — APPROVED** [review]
small typo then GTM

**Pull request merged**